.goreleaser.yml

```diff
@@ -57,8 +57,32 @@ builds:
       pre: make generate-web-assets
 
 nfpms:
-  - id: "influxdb2"
-    builds: ["influx", "influxd"]
+  - id: influx-cli
+    package_name: influx-cli
+    builds: [influx]
+    formats:
+      - deb
+      - rpm
+    bindir: /usr/bin
+    conflicts:
+      - influxdb
```

Comment:
I just want to check that this shouldn't be `influx` or `influxdb2`; otherwise I'm reading the semantics to be that influx-cli conflicts with 1.x, is that correct?

Reply:
Correct, conflicts with 1.x (since 1.x has its own `influx` binary).